.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: ci
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
push and pull_request should be sufficient, is there a reason to add workflow_dispatch here?